examples/compiler/02_data_regions.ipynb

```diff
@@ -11,15 +11,13 @@
    "cell_type": "markdown",
    "metadata": {},
    "source": [
-    "In this tutorial we will learn about data regions and how these impact the Operator construction. We will use the simple time marching example shown in the [01_iet](https://github.com/opesci/devito/blob/master/examples/compiler/01_iet.ipynb) tutorial. "
+    "In this tutorial we will learn about data regions and how these impact the `Operator` construction. We will use the simple time marching example."
```

Comment:
reminder:
>  We will use the simple time marching example."
... which one ? :)
probably you need to lift it from 03_iet...

Reply:
imho, i think that linking the something in the intro of the second notebook for references that live in the third notebook is somewhat confusing. (please note that the notebooks have shifted in order. `iet` is no longer the first notebook)

 also, the a time marching example is described in cell `[03]` of this very notebook

Reply:
ahhh I had missed this...

> also, the a time marching example is described in cell [03] of this very notebook

this sentence was confusing:

>  We will use the simple time marching example

you should probably rewrite it as

>  We will use a simple time-marching example

(or Operator instead of example doesn't matter)

Reply:
I mean, the article ('the' instead of 'a') was confusing

Reply:
of course, agreed